MyCryptoHQ/MyCrypto
# Token Balance try again button & equivalent values fix

Closes #1359 & closes #1425

### Description

Adds a retry button to token balances when they fail. Since I was testing with a wallet that was failing from what #1425 described, I went ahead and fixed that too.

### Changes

* Added a retry button when `state.wallet.tokensError` is set
* Refactored actions to use "refresh" as nomenclature instead of "set"
* Handled cases where CryptoCompare returns ludicrously high values for shitcoins
    * It was coming back with 1 ETH being worth 1e-8 HKG (Hacker Gold) which resulted in numbers that even BN couldn't handle.
* Fixed some missing translations

### Steps to Test

1. View the address `0xfbb1b73c4f0bda4f67dca266ce6ef42f520fbb98`
2. Confirm it loads (It didn't before.)
3. Keep refreshing and unlocking until it fails (I find that the MyCrypto node fails every once in a while)
4. Confirm you see "Try again" button, click it
5. Confirm it made another network request

### Screenshots

<img width="424" alt="screen shot 2018-04-05 at 4 16 48 pm" src="https://user-images.githubusercontent.com/649992/38389888-b931b6b2-38ed-11e8-957d-c7cf59c78af5.png">
